Cargo.toml

```
@@ -1,6 +1,6 @@
 [package]
 name = "redges"
-version = "0.2.1"
+version = "0.3.0"
```

Not a breaking change:
```suggestion
version = "0.2.2"
```